src/common/devtools.js

```diff
@@ -145,6 +150,25 @@ const report =
     )
   ];
 
+const snapshot =
+  (model) => {
+    const id = Date.now().toString(32)
+    const message =
+      [ `\n\n`
+        , `echo '${JSON.stringify(window.application.model.value)}' > `
+        , `./dist/snapshot-${id}.json`
+        , ` && `
+        , `servo -w -b --pref dom.mozbrowser.enabled http://localhost:6060\?replay\=snapshot-${id}.json`
```

Comment:
Up to you, but I think it's not useful  to dump "echo" and "servo". What about just dump the JSON?